ui/src/app/login/login.tsx

```diff
@@ -25,8 +26,9 @@ function getRedirect(): string {
 
 export function Login() {
     useCollectEvent('openedLogin');
+    const {t} = useTranslation();
```

Comment:
```suggestion
    const {t} = useTranslation();

    useCollectEvent('openedLogin');

```

same as [previous comment](https://github.com/argoproj/argo-workflows/pull/12682#discussion_r1495337679)